cdc/tests/.unmodify/processor_resolved_ts_fallback/run.sh

```diff
@@ -5,54 +5,40 @@ set -eu
 CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 source $CUR/../_utils/test_prepare
 WORK_DIR=$OUT_DIR/$TEST_NAME
-CDC_BINARY=cdc.test
+CDC_BINARY=tikv-cdc.test
 SINK_TYPE=$1
+UP_PD=http://$UP_PD_HOST_1:$UP_PD_PORT_1
+DOWN_PD=http://$DOWN_PD_HOST:$DOWN_PD_PORT
 
 function run() {
-	# TODO: kafka sink has bug with this case, remove this after bug is fixed
-	if [ "$SINK_TYPE" == "kafka" ]; then
-		return
-	fi
-
 	rm -rf $WORK_DIR && mkdir -p $WORK_DIR
 	start_tidb_cluster --workdir $WORK_DIR
 	cd $WORK_DIR
 
-	export GO_FAILPOINTS='github.com/tikv/migration/cdc/cdc/sink/SinkFlushDMLPanic=return(true);github.com/tikv/migration/cdc/cdc/sink/producer/kafka/SinkFlushDMLPanic=return(true)'
-	run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --logsuffix "1" --addr "127.0.0.1:8301" --pd "http://${UP_PD_HOST_1}:${UP_PD_PORT_1}"
+	export GO_FAILPOINTS='github.com/tikv/migration/cdc/cdc/sink/SinkFlushEventPanic=return(true)'
```

Comment:
`BeforeTiKVSinkFlushEvent=panic` ?

Maybe it would be better to change `failpoint.Inject("BeforeTiKVSinkFlushEvent", nil)` back as `failpoint.Inject("BeforeTiKVSinkFlushEventPanic", func() { log.Panic(...) })`. Then we can verify that the failpoint is injected successful by check the log. Refer to [processor_err_chan](https://github.com/tikv/migration/blob/main/cdc/tests/.unmodify/processor_err_chan/run.sh#L71).